base/database/setup.go

```diff
@@ -75,18 +73,17 @@ func loadEnvPostgreSQLConfig() *PostgreSQLConfig {
 		Passwd:   utils.Getenv("DB_PASSWD", "FILL"),
 		SSLMode:  utils.Getenv("DB_SSLMODE", "FILL"),
 
-		Timeout:               "60s",
-		ReadTimeout:           "60s",
-		WriteTimeout:          "60s",
-		MaxConnections:        250,
-		MaxIdleConnections:    50,
-		MaxConnectionLifetime: 60,
+		StatementTimeoutMs:     utils.GetIntEnvOrDefault("DB_STATEMENT_TIMEOUT_MS", 10000),
```

Comment:
This won't work. The system culling statements definitely take more than 10 seconds

Reply:
We can turn it of setting it to 0 (https://www.postgresql.org/docs/10/runtime-config-client.html) for some components, e.g. vmaas_sync or increase the value. But it's useful to avoid cumulating dead transactions in case that container is killed.

Reply:
Or we can turn it off by default (setting to `0`) and set timeouts only for some components (listener, evaluator).